<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deareator</class>
 <widget class="QDialog" name="Deareator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deareator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupLevel">
     <property name="title">
      <string>Level Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="5">
       <widget class="QPushButton" name="buttonLevelSetpointInc">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QPushButton" name="buttonLevelSetpointDec">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="9" column="5" colspan="3">
       <widget class="QLabel" name="labelDrainValve">
        <property name="text">
         <string>Drain Valve</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" rowspan="2" colspan="3">
       <widget class="QLineEdit" name="lineFlowValve"/>
      </item>
      <item row="12" column="0" colspan="3">
       <widget class="QLabel" name="labelDALevel">
        <property name="text">
         <string>DA Level</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5" colspan="3">
       <widget class="QLabel" name="labelLevelSetpoint">
        <property name="text">
         <string>Level Stepoint</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QPushButton" name="buttonFlowDec">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="10" column="5">
       <widget class="QPushButton" name="buttonDrainInc">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="12" column="5" colspan="2">
       <widget class="QLabel" name="labelHotwellLevel">
        <property name="text">
         <string>Hotwell Level</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="4">
       <widget class="QLabel" name="labelFlowValve">
        <property name="text">
         <string>Flow Valve</string>
        </property>
       </widget>
      </item>
      <item row="10" column="6" rowspan="2" colspan="2">
       <widget class="QLineEdit" name="lineDrainValve"/>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QPushButton" name="buttonLevelManual">
        <property name="text">
         <string>Manual</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="6" rowspan="2" colspan="2">
       <widget class="QLineEdit" name="lineLevelSetpoint"/>
      </item>
      <item row="11" column="5">
       <widget class="QPushButton" name="buttonDrainDec">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QPushButton" name="buttonLevelAuto">
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="buttonFlowInc">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="13" column="5" colspan="2">
       <widget class="QLineEdit" name="lineHotwellLevel"/>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QLineEdit" name="lineDALevel"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
